modules/kml/docs/api-reference/kml-loader.md

```diff
@@ -23,10 +23,10 @@ const data = await load(url, KMLLoader, options);
 
 ## Options
 
-| Option            | Type    | Default | Description                                                                                                |
-| ----------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------- |
-| `useLngLatFormat` | Boolean | `true`  | KML longitudes and latitudes are specified as `[lat, lng]`. This option "normalizes" them to `[lng, lat]`. |
-| `useColorArrays`  | Boolean | `true`  | Convert color strings to arrays.                                                                           |
+| Option          | Type    | Default     | Description                                                                                                |
+| --------------- | ------- | ----------- | ---------------------------------------------------------------------------------------------------------- |
+| `kml.normalize` | Boolean | `true`      | KML longitudes and latitudes are specified as `[lat, lng]`. This option "normalizes" them to `[lng, lat]`. |
```

Comment:
Should this be forced as `true` when output is not `raw`? Geojson spec requires longitude-latitude order

Reply:
Yes, we are giving the user the ability to shoot themselves in the foot. But this loader is so experimental I don't wan't to fiddle more with it now.